packages/hoppscotch-common/src/services/spotlight/searchers/environment.searcher.ts

```diff
@@ -289,6 +295,45 @@ export class SwitchEnvSpotlightSearcherService
     }
   )[0]
 
+  fetchTeamEnvironmentList(
+    teamID: string,
+    fn: (results: TeamEnvironment[]) => void
+  ) {
+    const results: TeamEnvironment[] = []
+
+    runGQLQuery({
+      query: GetTeamEnvironmentsDocument,
+      variables: {
+        teamID: teamID,
+      },
+    }).then(
```

Comment:
Can we stick with `async/await` wherever possible? We can also leverage the object shorthand syntax when the key is named the same as the variable holding the value.